### PR TITLE
Clone detection paper from SANER 2019

### PR DESCRIPTION
**Pull request by GeileHirnbude**
Add “Learning-Based Recursive Aggregation of Abstract Syntax Trees for Code Clone Detection” paper

Signed-off-by: Lutz Büch <lutz.buech@informatik.uni-heidelberg.de>

Please fill the `[ ]` with an `x` to make sure that you read and followed the checklist

- [x] This addition is about Machine Learning __on Code__ specifically. General links should be added to [Awesome Machine Learning](https://github.com/josephmisiti/awesome-machine-learning) instead.
- [x] I have checked that the proposed addition is not already in the awesome list.
- [x] I have signed-off my commits as detailed [here](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
- [x] The links follow the link template:

        * [Title](URL) - Description.
    or

        * [Title](URL) - Description [Code](link to code).
- [x] The addition is correctly classified as a paper, blog post, talk, software or dataset.
- [x] The commit message must start with a verb (`Add …`, `Deprecate …`) and end without a dot. For exemple:

        Deprecate Theano software

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 * <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [A Deep Learning Approach to Program Similarity](https://dl.acm.org/citation.cfm?id=3243131) - Niccolò Marastoni, Roberto Giacobazzi and Mila Dalla Preda, MASES 2018.
 * <img src="badges/6-pages-gray.svg" alt="6-pages" align="top"> [Recurrent Neural Network for Code Clone Detection](https://seim-conf.org/media/materials/2018/proceedings/SEIM-2018_Short_Papers.pdf#page=48) - Arseny Zorin and Vladimir Itsykson, SEIM 2018.
 * <img src="badges/8-pages-gray.svg" alt="8-pages" align="top"> [The Adverse Effects of Code Duplication in Machine Learning Models of Code](https://arxiv.org/abs/1812.06469) - Miltiadis Allamanis, 2018.
+* <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [Learning-Based Recursive Aggregation of Abstract Syntax Trees for Code Clone Detection](https://ieeexplore.ieee.org/document/8668039) - Lutz Büch and Artur Andrzejak, SANER 2019
```
Comment (bzz) [inline README.md]:
```suggestion
* <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [Learning-Based Recursive Aggregation of Abstract Syntax Trees for Code Clone Detection](https://ieeexplore.ieee.org/document/8668039) - Lutz Büch and Artur Andrzejak, SANER 2019.
```

Always a good idea to keep formatting consistent 😉

Comment (EgorBu) [reply README.md]: I can't download an article without registration.
Is it possible to upload it somewhere else?
I want all articles to be available without registration

**Review by EgorBu — CHANGES_REQUESTED** [review]
Link to download article shouldn't require registration

**Comment by bzz** [comment]
@GeileHirnbude thanks, can download now. 
Please also apply the consistent formatting, as suggested above.

**Review by bzz — APPROVED** [review]

**Review by EgorBu — APPROVED** [review]
Thank you!

**Pull request merged**